web-registry/src/pages/ProjectMetadata/ProjectMetadata.SelectedForm.tsx

```diff
@@ -17,16 +17,17 @@ export const ProjectMetadataSelectedForm = ({
   isVCS,
   metadata,
   submit,
-}: Props): JSX.Element =>
-  isVCS ? (
-    // TODO https://github.com/regen-network/regen-registry/issues/908
-    <Box sx={{ display: 'flex', justifyContent: 'center' }}>
-      {'VCS metadata form not implemented yet'}
-    </Box>
-  ) : (
-    <ProjectMetadataForm
-      submit={submit}
-      initialValues={metadata}
-      graphData={graphData}
-    />
-  );
+}: Props): JSX.Element => (
+  // isVCS ? (
+  //   // TODO https://github.com/regen-network/regen-registry/issues/908
+  //   <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+  //     {'VCS metadata form not implemented yet'}
+  //   </Box>
+  // ) : (
+  <ProjectMetadataForm
```

Comment:
✂️  ?

Reply:
left a TODO - had to do this temporarily because we are using the paste metadata form for all cases for now.